package.json

```diff
@@ -303,7 +301,7 @@
     "@types/mock-fs": "^4.10.0",
     "@types/moment-timezone": "^0.5.12",
     "@types/mustache": "^0.8.31",
-    "@types/node": ">=10.17.17 <10.20.0",
+    "@types/node": "14.17.32",
```

Comment:
Should we use a caret range here so that minor version bumps are accepted? That's what we have in the resolution.

Reply:
Yeah, I can add that